XenAdmin/Wizards/XenWizardBase.resx

```diff
@@ -198,6 +198,84 @@
   <data name="XSHelpButton.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
     <value>Right</value>
   </data>
+  <data name="XSHelpButton.Image" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">

```

Comment:
The image shouldn't be embedded in the resx file

Reply:
Yes. Interesting, where was it from....